src/Makefile

```diff
@@ -10,7 +10,7 @@ PROG    = $(PROGNAME)$(PROG_SUFFIX)
 # Will dynamically generate version.h with the build number.
 CFLAGS += -DHAVE_VERSION_H
 
-CFLAGS += -I. -fPIC -std=c99 -O0
+CFLAGS += -I. -fPIC -std=c99
 # Replace above line with the two below and then look at gmon.out
 # to do performance monitoring
 # CFLAGS += -g -pg -I. -fPIC -std=c99 -O0
```

Comment:
For completeness, also remove the -O0 ~~here~~ on line 16.

Reply:
Done